models/articles.model.js

```diff
@@ -11,7 +11,24 @@ function fetchArticleById(article_id) {
         });
 };
 
-function fetchArticles() {
+function fetchArticles(sort_by, order) {
+    if (!sort_by) {
+        sort_by = 'created_at';
+    }
+    if (!order) {
+        order = 'desc';
+    }
+    const validSortColumns = ['article_id', 'title', 'topic', 'author', 'created_at', 'votes'];
+    const validOrders = ['asc', 'desc'];
+
+    if (!validSortColumns.includes(sort_by)) {
+        return Promise.reject({ status: 400, msg: 'Invalid sort_by column' });
+    }
+
+    if (!validOrders.includes(order)) {
```

Comment:
It might be good to modify this to `!validOrders.includes(order.toLowerCase())` just in case someone types ASC or DESC.